.github/wiki/Standard-File-Organization.md

```diff
@@ -146,6 +164,10 @@ This folder contains the main sources of the project. There are three sub-folder
 
 This folder contains the unit testing and integration testing of the main project.
 
+## {ProjectName}KIFUITests
+
+This folder contains the KIF UI testing of the main project. Use KIF instead of UITests for speed and reliability.
```

Comment:
```suggestion
This folder contains the KIF UI testing of the main project. Use KIF instead of Xcode default UITests for speed and reliability.
```

or

```suggestion
This folder contains the KIF UI testing of the main project. Use KIF instead of XCTest for speed and reliability.
```